docs/testrunner-toolkit/configuration.md

```diff
@@ -1,177 +1,221 @@
 
 ```bash
-saucectl run --ccy 10
+# Only install dev dependencies
+npm install --only=dev
+
+saucectl run
 ```
 
-A setting of `10` runs up to 10 test suites at the same time. If the test has more suites than that, excess suites are queued and run in order at the completion of currently running suites.
+##### Uninstall Nonessential Dependencies
 
-When running on Sauce cloud, the maximum concurrency that you can use is limited by your account settings.
+If your standard install includes dependencies that aren't needed to run your tests, uninstall them prior to bundling.
 
-## Set Different Screen Resolutions
+```bash
+# Install node dependencies
+npm ci # or "npm install"
+
+# Remove unneeded dependencies
+npm uninstall appium
+npm uninstall express
+
+saucectl run
+```
 
-<p><small>supported frameworks: <Highlight color="#25c2a0">cypress</Highlight></small></p>
+##### Install Essential Dependencies Individually
 
-If you wish to execute tests on different screen resolutions while using Testrunner Toolkit, add the `screenResolution` parameter to your `.sauce/config.yml`:
+If you know that your tests require only specific dependencies, install them individually instead of running `npm install` or `npm ci`.
 
-__Example__:
+```bash
+# Install individual dependencies
+npm install cypress-xpath
+npm install @cypress/react
 
-```reference
-https://github.com/saucelabs/saucectl-cypress-example/blob/4141ba08e501ae2be691969a1626c78c18a77aff/.sauce/config.yml
+saucectl run
 ```
 
-## Additional Information
+#### Set NPM Packages in `config.yml`
+
+You can avoid installing or uninstalling dependencies prior to each bundling operation by defining a default set of NPM packages to install in your sauce configuration file using the `npm` parameter, as shown in the following example:
 
-* [Run a Test](/testrunner-toolkit/running-tests): Learn how to run tests using TestRunner.
-* [Pipeline Collaboration](/testrunner-toolkit/integrations) Run your tests in your CI platform:  
-    * [Jenkins](/testrunner-toolkit/integrations/jenkins)
-    * [GitHub Actions](/testrunner-toolkit/integrations/github-actions)
-* [Configuration Syntax](/testrunner-toolkit/configuration/common-syntax) Review the syntax and data fields for `saucectl` configuration settings that are common to all set ups, or review framework-sensitive configuration settings:
-    * [Cypress Syntax](/testrunner-toolkit/configuration/cypress)
-    * [Playwright Syntax](/testrunner-toolkit/configuration/playwright)
-    * [TestCafe Syntax](/testrunner-toolkit/configuration/testcafe)
-    * [Puppeteer Syntax](/testrunner-toolkit/configuration/puppeteer)
+```jsx title= "config.yml npm example"
+npm:
+  registry: https://registry.npmjs.org
+  packages:
+    lodash: "4.17.20"
+    "@babel/preset-typescript": "7.12"
+    "@cypress/react": "^5.0.1"
+```
+:::note
+The `registry` parameter is only supported in Cypress running in Docker.
```

Comment:
The `registry` parameter is supported in `cypress`, `playwright` and `testcafe` runner. And it works for sauce cloud as well for these 3 runners.

Reply:
Thanks!